.gitignore

```diff
@@ -8,4 +8,5 @@ Archive
 /reports/
 /generated/
 /install-jdk.sh
-.to
\ No newline at end of file
+.to
+docs/.jekyll-metadata
```

Comment:
docs already has its own .gitignore, so this should go there to keep all the docs related gitignore information together.